doc/changes/changes_0.2.5.md

```diff
@@ -12,10 +12,18 @@ Fixed broken link
 
 ## Dependency Updates
 
+### Compile Dependency Updates
```

Comment:
Did you update changelog of version 0.2.5 by accident?
I guess the changes should go to version 1.0.0

Reply:
Fixed